ta4j-core/src/test/java/org/ta4j/core/XlsTestsUtils.java

```diff
@@ -0,0 +1,107 @@
+/**
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2014-2017 Marc de Verdelhan & respective authors (see AUTHORS)
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy of
+ * this software and associated documentation files (the "Software"), to deal in
+ * the Software without restriction, including without limitation the rights to
+ * use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of
+ * the Software, and to permit persons to whom the Software is furnished to do so,
+ * subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS
+ * FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR
+ * COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER
+ * IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+ * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+package org.ta4j.core;
+
+import java.time.Duration;
+import java.time.Instant;
+import java.time.ZoneId;
+import java.time.ZonedDateTime;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+import org.apache.poi.hssf.usermodel.HSSFDateUtil;
+import org.apache.poi.hssf.usermodel.HSSFWorkbook;
+import org.apache.poi.ss.usermodel.Cell;
+import org.apache.poi.ss.usermodel.CellValue;
+import org.apache.poi.ss.usermodel.FormulaEvaluator;
+import org.apache.poi.ss.usermodel.Row;
+import org.apache.poi.ss.usermodel.Sheet;
+
+public class XlsTestsUtils {
+
+    public static Sheet getDataSheet(Class clazz, String xlsFileName) throws Exception {
+        // we assume that first sheet contains data
+        HSSFWorkbook workbook = new HSSFWorkbook(clazz.getResourceAsStream(xlsFileName));
+        Sheet sheet = workbook.getSheetAt(0);
+        return sheet;
+    }
+
+    public static void setParamValue(Sheet sheet, int paramIndex, double value) {
+        // first param is in B3 cell
+        // second param is in B4 cell
+        Row row = sheet.getRow(2 + paramIndex);
+        Cell cell = row.getCell(1);
+        cell.setCellValue(value);
+    }
+
+    public static TimeSeries readTimeSeries(Sheet sheet) {
+        // prices starts from 7th row until the end of sheet
+        // price data exists in first 5 columns (week date, open, high, low, close)
+        TimeSeries series = new BaseTimeSeries();
```

Comment:
I can add column in xls for volume - just in case, some indicators use it
But what is amount? I don't know any indicator which uses it. I don't think it is necessary.

Unfortunately I didn't know about these branching rules in this project. I will change this PR.